test/e2e/Tests/Fixtures/FixtureHelpers.cs

```diff
@@ -12,6 +12,8 @@ namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E;
 
 public static class FixtureHelpers
 {
+    private const string MSSQL_SA_PASSWORD = "IntentionallyExposedPassword1";
```

Comment:
Should we create a randomly generated password for every run so that we're not exposing a secret?

Reply:
Done - the pipeline will now generate a random GUID for the password. 
I also added a .runsettings file for the test project with the environment vars that would be necessary to configure the tests to run against MSSQL, so that locally you can debug these tests easier.